app/Http/Controllers/ChannelController.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace Suyabay\Http\Controllers;
+
+use Suyabay\Channel;
+use Suyabay\Http\Requests;
+use Illuminate\Http\Request;
+use Illuminate\Database\QueryException;
+use Suyabay\Http\Controllers\Controller;
+
+class ChannelController extends Controller
+{
+    protected $response;
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $channels = Channel::orderBy('id', 'asc')->paginate(10);
+
+        return view('dashboard.pages.view_channels', compact('channels'));
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function createIndex()
+    {
+        return view('dashboard.pages.create_channel');
+    }
+
+    /**
+     * Process channel creation
+     */
+    public function processCreate(Request $request)
+    {
+        try
+        {
+            $channel = Channel::create([
+                'channel_name'         => $request->name,
+                'channel_description'  => $request->description,
+                'subscription_count'   => 0
+            ]);
+            $this->response =
+            [
+                'message' => 'Channel created Successfully',
+                'status_code' => 200
+            ];
+        } catch (QueryException $e) {
+            $this->response =
+            [
+                'message' => 'Channel already exist',
+                'status_code' => 400
+            ];
+        }
+
+        return $this->response;
+    }
+
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        $channels = Channel::where('id', $id)->first();
+
+        return view('dashboard.pages.edit_channel', compact('channels'));
+    }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function update(Request $request)
+    {
+        try
```

Comment:
The opening brace should be on the same line as the `try` keyword. Check out the PSR-2 guide [http://www.php-fig.org/psr/psr-2/#5-6-try-catch](http://www.php-fig.org/psr/psr-2/#5-6-try-catch)